examples/api-routes/pages/api/people/[id].ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { people } from '../../../data'
+import { Person } from '../../../interfaces'
+
+type Error = {
+  message: string
+}
```

Comment:
This should probably be called something else to not clash with the native `Error` class.